teams/tidb/membership.json

```diff
@@ -2,14 +2,14 @@
     "_comment": "Items of lists in this file are in alphabetical order. Please keep it on modification.",
     "description": "",
     "maintainers": [
-        "XuHuaiyu",
```

Comment:
Please notice that we sort them in 'ASCII-alphabetical' order, so I think you don't need to reorder this.

Just like: https://github.com/pingcap/community/blob/b6895f306859e67b59a52b01b26262be43a3dbc8/teams/tidb/membership.json#L33-L41